ovn-fake-multinode-utils/scripts/log-collector.sh

```diff
@@ -9,11 +9,13 @@ for c in $(podman ps --format "{{.Names}}" --filter "name=${node_name}"); do
     mkdir ${host}/$c
     podman exec $c ps -aux > ${host}/$c/ps
     podman exec $c bash -c 'touch /tmp/process-monitor.exit && sleep 5'
+    podman exec $c bash -c 'ovs-ofctl dump-flows br-int > /tmp/open-flows.log'
```

Comment:
@dceara we will have file for each of the nodes, in this way we can just look at what it is meaningful. Do you prefer to define a subset of the node list?

Reply:
> I think it's useful to also dump groups:
> 
> ```
> ovs-ofctl dump-groups br-int
> ```

@dceara ack, I will fix it

Reply:
> @dceara we will have file for each of the nodes, in this way we can just look at what it is meaningful. Do you prefer to define a subset of the node list?

Yes, I'm afraid this will increase the collected log size significantly.  Let's collect flows/groups from max 10 nodes?  It should be relevant enough.

Reply:
@dceara ack, do you mean to collect info just from first 10 nodes?

Reply:
Yes, something like that.